en/docs/reference/connectors/mongodb-connector/3.x/mongodb-connector-example.md

```diff
@@ -0,0 +1,247 @@
+# MongoDB connector example
+
+The MongoDB Connector can be used to perform CRUD operations in the local database as well as in MongoDB Atlas (cloud version of MongoDB).
+
+## What you&#39;ll build
+
+This example explains how to use MongoDB Connector to insert and find documents from a MongoDB database.
+
+The sample API given below demonstrates how the MongoDB connector can be used to connect to the MongoDB Server and perform **insert many** and **find** operations on it.
+
+- `/insertmany`: The user sends a request payload that includes the connection information, collection name, and the documents to be inserted. This request is sent to the integration runtime by invoking the MongodbConnector API. This will insert the documents into the MongoDB database.
+
+    <p><img src="{{base_path}}/assets/img/integrate/connectors/mongodb-conn-1.png" title="Insert many function" width="800" alt="Insert many function" /></p>
+
+- `/find`: The user sends the request payload containing the connection information, collection name, and the find query. This request is sent to the integration runtime by invoking the MongodbConnector API. Once the API is invoked, it returns the documents matching the find query.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/mongodb-conn-2.png" title="Find function" width="800" alt="Find function"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Before you begin
+
+If you want to connect to MongoDB Atlas, follow the steps mentioned below to get the connection string.
+
+1. In the Clusters view, click **Connect** for the cluster to which you want to connect.
+
+2. Click **Choose a connection method**.
+
+3. Click **Connect your application**.
+
+4. Select Java from the **Driver** menu.
+
+5. Select the correct driver version from the **Version**(`4.3 or later`) menu.
+
+6. Clear the **Include full driver code example** check box to get the connection string.
+
+## Copy the MongoDB driver JAR files
+
+Copy the following JAR files to the `<MI_HOME>/dropins` directory.
+
+* [mongodb-driver-sync - 4.9.1](https://repo1.maven.org/maven2/org/mongodb/mongodb-driver-sync/4.9.1/mongodb-driver-sync-4.9.1.jar)
+* [mongodb-driver-core - 4.9.1](https://repo1.maven.org/maven2/org/mongodb/mongodb-driver-core/4.9.1/mongodb-driver-core-4.9.1.jar)
+* [bson - 4.9.1](https://repo1.maven.org/maven2/org/mongodb/bson/4.9.1/bson-4.9.1.jar)
+
+## Create the integration project
+
+1.  Create a new integration project named `MongodbConnector`.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/mongodb-conn-3.png" title="Create project" width="500" alt="Create project"/>
+
+2.  Click `+` on Extension panal APIs to create the REST API.
```

Comment:
Shall we rephrase this?

Reply:
Let's use the same terms we have used in the QSG to refer the UI components in VS Code